Python/Images_to_GIF/README.md

```diff
@@ -0,0 +1,22 @@
+# images_to_gif.py
+A simple Python script that takes a directory of images, resizes them, and combines them into a .gif file.
+
+## How to use
+python images_to_gif.py /fullpath/to/image/folder
+
+## Also this will work
+python images_to_gif.py
+> Enter fullpath to directory: /fullpath/to/image/folder
+
+## requirements
+imageio
+Pillow
+
+Pillow is a PIL fork.
```

Comment:
This section is not needed. Add an instruction to download all dependencies using `requirements.txt`

Reply:
Thanks for the review @HarshCasper , I believe the README conforms to the style guidelines now.

Reply:
@HarshCasper can this be merged now? better late than never :)